src/mscorlib/src/System/Buffer.cs

```diff
@@ -364,15 +364,18 @@ internal unsafe static void Memmove(byte* dest, byte* src, nuint len)
             return;
 
             MCPY05:
+#if !ARM64
             // PInvoke to the native version when the copy length exceeds the threshold.
             if (len > CopyThreshold)
             {
                 goto PInvoke;
             }
-
             // Copy 64-bytes at a time until the remainder is less than 64.
             // If remainder is greater than 16 bytes, then jump to MCPY00. Otherwise, unconditionally copy the last 16 bytes and return.
             Debug.Assert(len > 64 && len <= CopyThreshold);
```

Comment:
Not: I would delete CopyThreadshold part of the assert to avoid the `#else` part of the ifdef. It has no value because of it just duplicates check a few lines above.